### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by starocketeer**
Ready

### CODE REVIEW

**Comment by starocketeer** [comment]
Ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use sqlite3 as the database for Active Record
```
Comment (saobv-2223) [inline Gemfile]: remove all the comment lines and using " instead of '

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: in js and css file, we use ' instead of "

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: in ruby file, we use " instead of '

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: using "

File: config/credentials.yml.enc [hunk 5]
```diff
@@ -0,0 +1 @@
+6UtdL20nDcU/h1e/s+cLuMV8OKULpWdob9Q8T9seSN5CkUGXv/xFIh46S/NSF2b+yQ/r2jhDpp8pJv0TKcm7l75CMmpGnWdlN4Hrb/9n8dZFXSuOtkCyOqzInH1QxolBRxllD3Z4Dntmd6fLd9q2ObKYklK1uPHQWLj7x4cydkOq7o560GNhfx/FrRXH7fzcqijyeO0C8VFPWwfoS9ixVaxAt7xIx11B86+RvJIiYY8tD6IxxOucQAH8gjpfQOnnXI9EhGX3b9pQWr7fSAbfzAc1scoRxvm/7NNXvY6OFNlsrHKNBHrkoi5oRlq1xGpC2hnfRDjSBVaCXHsjR8SUy3/G7sPlPGI9alMakKkG3HYiPmJ9HH+P7hqpw7JnCMIKQh2dp+4T9FDyQd/3XClzmeDih3emK0uDIs+x--D9NGUrjckhCra6rx--3o7vuYqG5aXSFwk3xuE5fA==
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: create `credentials.yml.enc.example` and push to git, this file you add to gitignore

File: config/database.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
```
Comment (saobv-2223) [inline config/database.yml]: do same as `credentials.yml.enc` file

File: test/application_system_test_case.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
```
Comment (saobv-2223) [inline test/application_system_test_case.rb:2]: we don't use test folder, so you add it to gitignore. Please

File: Gemfile (hunk 1 above)
Comment (starocketeer) [reply Gemfile]: done

File: app/javascript/packs/application.js (hunk 2 above)
Comment (starocketeer) [reply app/javascript/packs/application.js]: done

File: app/mailers/application_mailer.rb (hunk 3 above)
Comment (starocketeer) [reply app/mailers/application_mailer.rb]: done

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (starocketeer) [reply app/views/layouts/application.html.erb]: done

File: config/credentials.yml.enc (hunk 5 above)
Comment (starocketeer) [reply config/credentials.yml.enc]: done

File: config/database.yml (hunk 6 above)
Comment (starocketeer) [reply config/database.yml]: done

**Comment by starocketeer** [comment]
Ready

File: Gemfile [hunk 8]
```diff
@@ -0,0 +1,34 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.1"
+
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+gem "sqlite3", "~> 1.4"
```
Comment (ngocvt-0484) [inline Gemfile:7]: please use gem mysql2
remove gem sqlite3

**Pull request merged**